Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -38,7 +38,84 @@ public static class TextTable
         /// </summary>
         public static string Build(string message, int padding)
         {
-            return "";
+            if (message.Length == 0)
+            {
+                return ""; 
+            }
+            // make a array so I can find the longest word. 
+
+            var arrayOfWords = message.Split(Environment.NewLine);
+
+            // Find length of longest word 
+           var longestWord = ""; 
+
+            if (arrayOfWords.Length == 1)
+            {
+                longestWord = arrayOfWords[0]; 
+            }
+            else
+            {
+                switch (arrayOfWords[0].CompareTo(arrayOfWords[1]))
+                {
+                    case -1:
+                        longestWord = arrayOfWords[1]; 
+                        break;
+                    default:
+                        longestWord = arrayOfWords[0]; 
+                        break;
+                }
+            }
+
+            // print table 
+
+            // print header
```

Comment:
All code below is highly inefficient and refactoring them to be functions would be great!

Reply:
is it then not a bad thing that I use the stringbuilder of the table as argument and later after the  change as return value all the time ?